### ISSUE TITLE
The issue with configuration on the Cityscapes dataset

### ISSUE THREAD

**Issue by syqisme** [williamium3000/diverse-cotraining#3]
Hello, I have recently been working on your project and have had good results with the Pascal VOC dataset. However, the performance on the Cityscapes dataset has been poor. I have carefully studied the configuration files you provided but haven't been able to resolve the issue. Therefore, I wanted to inquire if there are any specific considerations I should keep in mind regarding this dataset?

I have adjusted the loss function to OHEM loss and changed the evaluation method to sliding window evaluation with 19 categories. I also made modifications to the learning rate. Are there any other aspects I should pay attention to? I'm quite puzzled by this issue at this stage and don't know how to proceed. Could you provide me with some advice? I would be extremely grateful for any help you could offer!

**Comment by williamium3000** [comment]
Can you share your log files. Or you can compare your log files with mine. I have posted some logs here https://github.com/williamium3000/diverse-cotraining/tree/main/logs/cityscapes

**Comment by syqisme** [comment]
> Can you share your log files. Or you can compare your log files with mine. I have posted some logs here https://github.com/williamium3000/diverse-cotraining/tree/main/logs/cityscapes


Hello author,

Thank you very much for your response. Regarding the issue, I have managed to fix it by adjusting the parameters in the past few days. I am a beginner in semi-supervised learning and just started my first year of graduate school this year. Your work has provided me with significant inspiration and assistance in my studies. Thank you!
